keylime/cloud_verifier_common.py

```diff
@@ -158,7 +158,14 @@ def process_quote_response(agent, ima_policy, json_response, agentAttestState) -
     agentAttestState.set_boottime(boottime)
 
     ima_keyrings = agentAttestState.get_ima_keyrings()
-    tenant_keyring = file_signatures.ImaKeyring.from_string(agent["ima_sign_verification_keys"])
+
+    # If ima_sign_verification_keys was provided to agent by tenant directly, use that. Otherwise, find keyring in IMA policy.
+    if agent["ima_sign_verification_keys"]:
+        verification_key_string = agent["ima_sign_verification_keys"]
```

Comment:
We should add a deprecation warning to this.

Reply:
Done - I added a comment in the verifier, but the tenant will also print a message when the `ima_sign_verification_keys` argument is provided.